### DISCUSSION TITLE
How i can implement teams -> roles with checkboxes or dependant fields ?

### DISCUSSION THREAD

**Discussion by lucacalcaterra** [filamentphp/filament#1831] [Help]
Hi,

i'm developing a little app .

I managed the permissions/roles/teams with laratrust (should be quite the same with spatie permission) with attachform when edit user.

It's possible to implement this simpler with checkbox ? Should be as in figure but can manage manyToMany on same form (I.e. All user's roles for every team)

I'm not very expert on laravel programming and i'm doing too much confusion, so hope you can guide me the wrong direction.
Thanks !

I attach the screenshoots so you can understand .

Thanks !!

![image](https://user-images.githubusercontent.com/879825/157293886-2afac510-5aa1-448b-bfa5-15d6b21092e3.png)

**Comment by danharrin** [comment]
You should use a Teams relation manager and have the roles field inside there? I wouldn't recommend cramming in long lists of roles for each team in a form like that.

**Reply by lucacalcaterra** [reply]
I solved with a dedicated panel with form attach. Thanks !

**Reply by parijke** [reply]
Can you elaborate a bit more (preferably with code example?)
